### PR TITLE
Add home key for magic-namespace

### PR DESCRIPTION
**Pull request by AdamDang**
The key "home" is needed for ci testing, it's missing in this yaml. That sometimes will cause testing failure.

### CODE REVIEW

File: stable/magic-namespace/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 apiVersion: v1
 name: magic-namespace
-version: 0.1.0
+version: 0.1.1
 appVersion: 2.8.1
+home: https://www.mediawiki.org
```
Comment (timstoop) [inline stable/magic-namespace/Chart.yaml]: I'm stumped as to where you found this reference?

Comment (AdamDang) [reply stable/magic-namespace/Chart.yaml]: Hi @timstoop , I checked from google. I found some home keys use the same url as code source, some use the real home page url. Any way, a home key with any value will make the ci testing OK. I think it's good to have one value for this. The maintainer can correct this later. What's your idea about this? Just leave it empty?

Comment (timstoop) [reply stable/magic-namespace/Chart.yaml]: No sorry, I'm not okay with this. Upstream for this chart is the Kubernetes Helm project, actually, according to the image. You can link to that. Let's make sure that information that's added is actually correct.

**Review by timstoop — CHANGES_REQUESTED** [review]

File: stable/magic-namespace/Chart.yaml (hunk 1 above)
Comment (AdamDang) [reply stable/magic-namespace/Chart.yaml]: Thanks @timstoop 
In the values.yaml: repository: gcr.io/kubernetes-helm/tiller
Should the value for home key be "https://www.gcr.io/kubernetes-helm/"?
This site is unvisitable in China for the firewall reasons.

**Comment by timstoop** [comment]
@krancour Could you advise here, please?

**Comment by krancour** [comment]
I don't think I understand how this affects CI. What requirement is there that `Chart.yaml` must include that field or that it must be accessible from China (or anywhere, for that matter)?

If it _must_ be added, a link to a random site is certainly inappropriate. A link to the Helm site woukd be ok, but still not ideal. The point of this chart isn't to merely install a tiller per namespace. (Indeed, it does by default, but it's still optional.) The real objective of this chart is to provide a comprehensive strategy for cluster operators to manage team namespaces. Linking to Helm docs would overemphasize the tiller aspect, I think.

But let's start with understanding why we need home defined at all...

**Comment by AdamDang** [comment]
@krancour @timstoop Thanks for your advices to get me understanding about the home key. My procedure is as following:
1. Check from README.md, mostly I can find a the chart name with hyper link, I'll use the link as home URL. ----Not found for this chart.
2. Check from Internet and try to visit and confirm.----Not suggested
3. Otherwise, I use the same URL as source in the YAML file.---No source for this chart
So, According to your suggestion, I've modified the home to helm.

**Comment by krancour** [comment]
I still do not understand why you believe this key _must_ be set. Without some justification for that, I do not approve of this PR.

**Comment by unguiculus** [comment]
@krancour See #4161

**Comment by AdamDang** [comment]
@krancour According to the #4161, the home key is a checking item in the chart.yaml file. 
Do you need any more information to prove this PR? Thanks.

**Comment by AdamDang** [comment]
@krancour 
I think the root cause is clear. I will close this PR and create a new one.Thanks. 
BTW: I've checked all the charts directories and add the home keys  and appVersion keys
for those do not have the keys, except some chart that has problem in e2e tests.

**Pull request closed without merging**